presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -74,6 +74,7 @@
     public static final String LEGACY_ORDER_BY = "legacy_order_by";
     public static final String ITERATIVE_OPTIMIZER = "iterative_optimizer_enabled";
     public static final String ITERATIVE_OPTIMIZER_TIMEOUT = "iterative_optimizer_timeout";
+    public static final String USE_NEW_STATS_CALCULATOR = "use_new_stats_calculator";
```

Comment:
now need for use of `use_`

Reply:
Just `new_stats_calculator` doesn't sound right. Any other ideas?

Reply:
hm... why do we care so much about this coefficient based calculator?

Reply:
Maybe just `use_stats_calculator`, to make a switch to disable stats calculation in case of bug. So user could run a query without CBO.

Reply:
This would be bigger a change. I think we should ultimately remove `CoefficientBasedStatsCalculator` (on my todo) -- however, for that we may need to introduce a feature toggle like you said (`use_stats_calculator` or something) to suppress stats calculation in `EXPLAIN`. For now, I would prefer not to do this.

Reply:
`enable_new_stats_calculator` ?

Reply:
sounds good